templates/robotics/index.html

```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
     <hr class="p-separator" />
   </div>
   <div class="row">
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
+    <div class="col-6 u-hide--small u-align--center u-vertically-center">
       {{
         image(
-        url="https://assets.ubuntu.com/v1/9c80823b-confinement.svg",
+        url="https://assets.ubuntu.com/v1/6eb072a2-ESM+for+ROS_9.svg",
         alt="",
-        height="200",
-        width="350",
+        height="202",
+        width="200",
         hi_def=True,
         ) | safe
       }}
     </div>
     <div class="col-6">
-      <h3>Each package is its own sandbox</h3>
-      <p>Each application running on your robot is confined to its own environment, adding an additional layer of security. Anomalies will never propagate through the system. The total isolation of processes and their associated data makes localisation and resolution of errors easy.</p>
+      <h3>Robotics security</h3>
+      <h4>Comply with cybersecurity requirements with in-built security for your device</h4>
+      <p>With Ubuntu Core, in-built security means:</p>
+      <ul>
+        <li><a href="/core/features/secure-boot">Secure boot</a></li>
+        <li><a href="/engage/iot-disk-encryption">Full disk encryption</a></li>
+        <li><a href="https://snapcraft.io/docs/assertions">Signed software packages</a></li>
+        <li><a href="/core/features/recovery">Device recovery</a></li>
+        <li>10-year security update commitment</li>
+      </ul>
+      <p>Snap confinement keeps your apps separated and controls access to hardware.</p>
+      <p>Curate your IoT App Store from global suppliers, with constant application security scanning and updates.</p>
```

Comment:
"[IoT App Store](https://ubuntu.com/internet-of-things/appstore)" should be a link